File: app/src/main/java/nz/co/codebros/quakesnz/detail/QuakeDetailView.java
package nz.co.codebros.quakesnz.detail;

import nz.co.codebros.quakesnz.model.Feature;

/**
 * Created by leandro on 7/07/16.
 */
interface QuakeDetailView {
    void share(Feature publicId);

    void showDetails(Feature feature);

    void showLoadingError();
}

File: app/src/main/java/nz/co/codebros/quakesnz/detail/QuakeDetailView.kt
package nz.co.codebros.quakesnz.detail

import nz.co.codebros.quakesnz.core.data.Feature

/**
 * Created by Leandro on 16/02/2018.
 */
interface QuakeDetailView {
    fun updateFeature(feature: Feature)
    fun showLoadingError()
}